src/nvim/ex_cmds2.c

```diff
@@ -1339,6 +1339,20 @@ void dialog_changed(buf_T *buf, int checkall)
   }
 }
 
+/// Ask the user whether to unload the terminal buffer or not.
+///
+/// @param buf
+bool dialog_unload_terminal(buf_T *buf) {
+  char_u buff[DIALOG_MSG_SIZE];
+
+  dialog_msg(buff, _("Unload \"%s\"?"),
```

Comment:
Should we clarify "Kill and unload", or something like that?

Reply:
Or maybe "Closing", since that is a term we often use with terminals? Like `TermClose` etc.?